src/snowflake/cli/plugins/nativeapp/teardown_processor.py

```diff
@@ -143,18 +141,17 @@ def drop_application(
                     f"The following objects are owned by application {self.app_name}:\n{application_objects_str}"
                 )
             elif interactive:
-                if interactive:
```

Comment:
this `if` check is redundant since it's already within an `elif interactive:` block without `interactive` being mutated in between the checks